bin.src/reportPerformance.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from __future__ import division, print_function, absolute_import
+
+import argparse
+import os.path
+
+from lsst.utils import getPackageDir
+from lsst.validate.drp import report_performance
+
+description = """
+Report the performance vs. SRD and specified metrics for a JSON cache of a job.
+
+Produces results to:
+OUTFILE
+    RST summary table suitable for release report
+
+Example call:
+reportPerformance.py tests/CfhtQuick_output_r.json test_report_performance.rst \\
+    --release_metrics etc/release_metrics.yaml --release_level FY18
+"""
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description=description,
+                                     formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument('json_files', nargs='+',
+                        help='Paths to a JSON serialization of Job.  Separated with spaces.')
+    parser.add_argument('--output_file', default='report_performance.rst',
+                        help='The output RST filepath to write to.')
+    parser.add_argument('--srd_metrics',
+                        default=os.path.join(getPackageDir('validate_drp'),
+                                             'etc', 'metrics.yaml'),
+                        help='Path of YAML file with LPM-17 metric definitions.')
+    # 2017-07-21: MWV
+    # I'm not enforcing the value of 'srd_level' with a restricted 'choices'
+    # because I want to preserve the ability for somewhat arbitrary levels,
+    # even in the SRD for increased general utility of the function
+    # The acceptable values of 'choices' are defined by what's available
+    # in the 'srd_metrics' file.
```

Comment:
I agree this is a bit tricky.  What happens if someone specifies a level that is not in the file?  If the worst that happens is that no srd metrics end up in the report, I think that's o.k.